packages/cli-repl/src/cli-repl.ts

```diff
@@ -10,27 +10,33 @@ import isRecoverableError from 'is-recoverable-error';
 import { MongoshInternalError, MongoshWarning } from '@mongosh/errors';
 import { changeHistory, retractPassword } from '@mongosh/history';
 import { REPLServer, Recoverable } from 'repl';
-import jsonParse from 'fast-json-parse';
```

Comment:
Quoting the README of https://github.com/mcollina/fast-json-parse:

> This assumption holds true up to Node 6, from Node 7 and forward this module is not useful anymore.

Reply:
I mean is also only called once, even if it was faster i'm not sure we need to be that fast :)